test-suite/js/src/test/scala/org/scalajs/testsuite/javalib/lang/SystemJSTest.scala

```diff
@@ -85,29 +85,6 @@ class SystemJSTest {
 
     assertEquals("testtag.value", get("scalajs.testsuite.testtag"))
 
-    val compliantAsInstanceOf = get("scalajs.compliant-asinstanceofs") == "true"
-    if (compliantAsInstanceOf) {
-      try {
-        Int.box(5).asInstanceOf[String]
-        fail("Invalid class cast succeeded in presence of " +
-            "\"compliant-asinstanceofs\" flag.")
-      } catch {
-        case _: ClassCastException => // As expected
-      }
-    }
-    assertEquals(compliantAsInstanceOf, Platform.hasCompliantAsInstanceOfs)
```

Comment:
It's a bit sad that those tests are going away. They were useful to assert that the `linkingInfo` were indeed populated correctly, and that they would agree with the flags independently computed by the build.

I'm not sure what else to do, though.